elem/generate.go

```diff
@@ -178,7 +178,7 @@ func capitalize(s string) string {
 func descToComments(desc string) string {
 	c := ""
 	length := 80
-	for _, word := range strings.Split(desc, " ") {
+	for _, word := range strings.Fields(desc) {
```

Comment:
For reference, my initial solution here was:

```diff
 for _, word := range strings.Split(desc, " ") {
+	if word == "" { // Empty words can result from double spaces.
+		continue
+	}
 	if length+len(word)+1 > 80 {
```

But then I came up with this simpler solution of just using `strings.Fields`. The results were identical.